docs/getting-started-guides/scratch.md

```diff
@@ -0,0 +1,727 @@
+}
+```
+
+If you have selected the `--register-node=true` option for kubelets, they will now being self-registering with the apiserver.
+You should soon be able to see all your nodes by running the `kubect get nodes` command.
+Otherwise, you will need to manually create node objects.
+
+### Scheduler
+
+*TODO*: convert to version v1.
+
+Complete this template for the scheduler pod:
+```
+{
+"apiVersion": "v1beta3",
+"kind": "Pod",
+"metadata": {"name":"kube-scheduler"},
+"spec":{
+"hostNetwork": true,
+"containers":[
+    {
+    "name": "kube-scheduler",
+    "image": "$SCHEDULER_IMAGE",
+    "command": [
+                 "/bin/sh",
+                 "-c",
+                 "/usr/local/bin/kube-scheduler --master=127.0.0.1:8080"
+               ],
+    "livenessProbe": {
+      "httpGet": {
+        "path": "/healthz",
+        "port": 10251
+      },
+      "initialDelaySeconds": 15,
+      "timeoutSeconds": 15
+    },
+    }
+],
+}}
+```
+Optionally, you may want to mount `/var/log` as well and redirect output there.
+
+Start as described for apiserver.
+
+### Controller Manager
+To run the controller manager:
+  - select the correct flags for your cluster
+  - write a pod spec for the apiserver using the provided template
+  - start the scheduler pod
+
+Flags to consider using with controller manager.
+ - `--cluster_name=$CLUSTER_NAME`
+ - `--cluster-cidr=`
+   - *TODO*: explain this flag.
+ - `--allocate-node-cidrs=`
+   - *TODO*: explain when you want controller to do this and when you wanna do it another way.
+ - `--cloud_provider=` and `--cloud_config` as described in apiserver section.
+ - `--service_account_private_key_file=/srv/kubernetes/server.key`, used by [service account](../service_accounts.md) feature.  
+ - `--master=127.0.0.1:8080`
+
+Template for controller manager pod:
+```
+{
+"apiVersion": "v1beta3",
+"kind": "Pod",
+"metadata": {"name":"kube-controller-manager"},
+"spec":{
+"hostNetwork": true,
+"containers":[
+    {
+    "name": "kube-controller-manager",
+    "image": "$CNTRLMNGR_IMAGE",
+    "command": [
+                 "/bin/sh",
+                 "-c",
+                 "/usr/local/bin/kube-controller-manager $ARGS"
+               ],
+    "livenessProbe": {
+      "httpGet": {
+        "path": "/healthz",
+        "port": 10252
+      },
+      "initialDelaySeconds": 15,
+      "timeoutSeconds": 15
+    },
+    "volumeMounts": [
+        { "name": "srvkube",
+        "mountPath": "/srv/kubernetes",
+        "readOnly": true},
+        { "name": "etcssl",
+        "mountPath": "/etc/ssl",
+        "readOnly": true},
+      ]
+    }
+],
+"volumes":[
+  { "name": "srvkube",
+    "hostPath": {
+        "path": "/srv/kubernetes"}
+  },
+  { "name": "etcssl",
+    "hostPath": {
+        "path": "/etc/ssl"}
+  },
+]
+}}
+```
+
+
+### Logging
+
+**TODO** talk about starting Logging.
+
+### Monitoring
+
+**TODO** talk about starting Logging.
+
+### DNS
+
+**TODO** talk about starting DNS.
+
+### Miscelaneous Resources
```

Comment:
Miscellaneous is spelled (and in the index) wrong and this title sounds like a section where I can get more information beyond this guide.